tests/test_invoices.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+from dolib.client import Client
+
+
+@pytest.mark.vcr()
+@pytest.mark.block_network()
+def test_crud_invoce() -> None:
+    client = Client(token="fake_token")
+
+    # list all invoces
+    invoces = client.invoices.all()
+    assert len(invoces) > 0
+
+    # read invoice
+    invoce = client.invoices.get(invoces[0].invoice_uuid)
+    assert invoce.invoice_uuid == invoces[0].invoice_uuid
+
+    # read items
+    items = client.invoices.items(invoces[0].invoice_uuid)
+    assert len(items) > 0
```

Comment:
I would split this test so that you don't need comments but test names:

```suggestion
import pytest
from dolib.client import Client


@pytest.fixture
def client() -> Client:
    return Client(token="fake_token")


@pytest.mark.vcr()
@pytest.mark.block_network()
def test_list_all_invoices(client: Client) -> None:
    invoces = client.invoices.all()
    assert len(invoces) > 0


@pytest.mark.vcr()
@pytest.mark.block_network()
def test_read_invoice(client: Client) -> None:
    invoce = client.invoices.get(invoces[0].invoice_uuid)
    assert invoce.invoice_uuid == invoces[0].invoice_uuid


@pytest.mark.vcr()
@pytest.mark.block_network()
def test_read_items(client: Client) -> None:
    items = client.invoices.items(invoces[0].invoice_uuid)
    assert len(items) > 0
```

Reply:
ok, but this split and cassettes/test_invoices/test_crud_invoce.yaml to many files )

Reply:
Oh, crap